tests/acceptance/01_vars/01_basic/os_version_minor_simple.cf

```diff
@@ -0,0 +1,40 @@
+body common control
+{
+  bundlesequence => { "check" };
+}
+
+bundle agent check
+{
+  classes:
+    ubuntu_20_04|ubuntu_22_04|ubuntu_24_04::
+      "ok"
+        if => strcmp("$(sys.os_version_minor)", "04");
+      "not_ok"
+        if => not(strcmp("$(sys.os_version_minor)", "04"));
```

Comment:
The `ok` and `not_ok` classes are redundant since they test the exact same thing. It's like saying "if x is true and not false". Think about it, "if x is true, this implies that x is not false". So instead of testing `x and not x`, you can test only `x`